server/src/main/proto/spine_examples/kanban/events.proto

```diff
@@ -54,50 +55,116 @@ message BoardInitialized {
     BoardId board = 1 [(required) = true];
 }
 
-// A column placed on the board.
-message ColumnPlaced {
+// A request to add a new column to the board was made.
+message ColumnAdditionRequested {
 
-    // The ID of the board.
-    BoardId board = 1 [(required) = true];
+    // The ID of the column to add.
+    ColumnId column = 1 [(required) = true];
+
+    // The ID of the board to which to add the column.
+    BoardId board = 2 [(required) = true];
+
+    // The name of the column.
+    string name = 3 [(required) = true];
+
+    // The desired position of the column.
+    ColumnPosition desired_position = 4 [(required) = true];
+}
+
+// A column was created.
+message ColumnCreated {
+
+    // The ID of the created column.
+    ColumnId column = 1 [(required) = true];
+
+    // The ID of the board on which the column was created.
+    BoardId board = 2 [(required) = true];
+
+    // The name of the created column.
+    string name = 3 [(required) = true];
+}
+
+// A column was placed on a board.
+message ColumnPlaced {
 
     // The ID of the placed column.
-    ColumnId column = 2 [(required) = true];
+    ColumnId column = 1 [(required) = true];
+
+    // The ID of the board where column was placed.
+    BoardId board = 2 [(required) = true];
+
+    // The position on the board where the column was desired to be placed.
+    ColumnPosition desired_position = 3 [(required) = true];
+
+    // The position on the board where the column was actually placed.
+    ColumnPosition actual_position = 4 [(required) = true];
 }
 
-// A new card is created.
-message CardCreated {
-    option (is).java_type = "BoardAwareEvent";
+// A column was moved on a board.
+//
+// The `Board` emits this event to reflect the change of its column order and notify
+// subscribed entities and the `Column` that the column was moved.
```

Comment:
Let's discuss the difference one more time. Vocally.

And we do not speak in technical terms here, unless absolutely necessary. To me, this part hardly explains the difference between the events, but adds more confusion.